tests/test_istio.py

```diff
@@ -25,7 +25,7 @@ def test_istio(self):
         Sets up and validate istio.
         """
         print("Enabling Istio")
-        microk8s_enable("istio")
+        microk8s_enable("istio", 300, optional_args="--istio-version 1.20.0")
         print("Validating Istio")
         self.validate_istio()
```

Comment:
It would be great to also add a `verify_version` string kwarg to `validate_istio` so it can also explicitly verify that the installed version is correct?